e2e/support/helpers/e2e-downloads-helpers.js

```diff
@@ -110,15 +111,15 @@ export function assertSheetRowsCount(expectedCount) {
 
 function getEndpoint(fileType, questionId, publicUid, dashcardId, dashboardId) {
   if (dashcardId != null && dashboardId != null) {
-    return `api/dashboard/${dashboardId}/dashcard/${dashcardId}/card/${questionId}/query/${fileType}`;
+    return `api/dashboard/${dashboardId}/dashcard/${dashcardId}/card/${questionId}/query/${fileType}**`;
   }
 
   if (publicUid) {
     return `/public/question/${publicUid}.${fileType}**`;
   }
 
-  const questionEndpoint = `/api/card/${questionId}/query/${fileType}`;
-  const queryEndpoint = `/api/dataset/${fileType}`;
+  const questionEndpoint = `/api/card/${questionId}/query/${fileType}**`;
+  const queryEndpoint = `/api/dataset/${fileType}**`;
```

Comment:
This will make `cy.intercept` catch these URLs when they have a new `format_row=true/false` query parameter